common/djangoapps/course_about/api.py

```diff
@@ -10,10 +10,13 @@
 import logging
 from django.conf import settings
 from django.utils import importlib
+from django.core.cache import cache
 from course_about import errors
 
 DEFAULT_DATA_API = 'course_about.data'
 
+COURSE_INFO_API_CACHE_PREFIX = 'course_info_api_'
```

Comment:
Nitpick, but we've been using the name "Course About API" instead of "Course Info API" everywhere else, so we should stick with that for the variable names as well.